### ISSUE TITLE
Regarding combining hierarchical and multilabel classification

### ISSUE THREAD

**Issue by spartian** [AlexGidiotis/Advanced-ML-techniques#1]
Hello,

You have written multi-label and hierarchical classification in separate files. Is there a way to combine them? Like combining multi-label and hierarchical classification ?

**Comment by AlexGidiotis** [comment]
@spartian yes you absolutely can do this but it was not in the scope of this project.
